packages/blade/src/components/ProgressBar/_decisions/decisions.md

```diff
@@ -0,0 +1,104 @@
+| showPercentage | `boolean` | `true` | Control whether the progress bar should show the current progress percentage or not. Percentage would be automatically hidden when the progress bar is in an indeterminate state.  |
+
+### Sample Usage
+```jsx
+import { ProgressBar } from '@razorpay/components';
+
+<ProgressBar 
+  label='Loading' 
+  value={20} 
+  contrast='low'
+  size='large'
+  accessibilityLabel='Loading Payments'
+/>
+```
+
+### Accessibility
+- We'll accept a prop `accessibilityLabel` which will be passed on as `aria-label` for web & `accessibilityLabel` for native to aide screen readers
+- By default the value of `accessibilityLabel` will be the same as `label` prop with a fallback to `"Loading"` and can be overwritten with something more meaningful by the consumer.
+- We will set `aria-role` as `progressbar`
+- We will pass the value of `value` prop as `aria-valuenow`
+- Pass `aria-busy` as `true`
+
+## Handling pulsating animation
+After `3000ms` the progress bar would start pulsating to indicate to the user that the process is slow but isn't stuck.
+<img src="./progress-bar-pulse.gif" width="380" />
+
+**Problem:**
+- We have use-cases where a progress bar could be used:
+  - <img src="./progress-bar-static.png" width="240" />
+
+- In such cases, we wouldn't want the progress bar to pulsate after `3000ms` since this will remain static.
+
+**Possible Solutions:**
+1. Use prop `variant` as `'static'` and `'default' | 'animated'`
+
+- Pros:
+   - `variant` is a commonly used prop and makes it easy to understand that a different variation of the component exists
+
+- Cons:
+   - When the variant would be `static`, it won't entirely be static. When the `value` is changed, it will animate (easing with duration for filling the bar) to the new value. Only the pulsating animation would be disabled.
+   - Education effort would be needed to communicate exactly what we mean by a static variant and where should it be used.
+
+2. Use prop `animation` as `'none'` and `'default' | 'pulse'`
+
+- Pros:
+   - `animation` is more specific than `variant` since the only change would be related to animation.
+
+- Cons:
+  - When the animation would be `none`, it won't entirely be none. When the `value` is changed, it will animate (easing with duration for filling the bar) to the new value. Only the pulsating animation would be disabled.
+  - Education effort would be needed to communicate exactly what we mean by animation as none, pulse (or default) and where should it be used.
+  
+3. Use prop `showPulsatingAnimation` as `boolean`
+
+- Pros:
+   - It would be a single-purpose boolean and is easy to understand what it does.
+
+- Cons:
+   - Could be slightly misleading since `showPulsatingAnimation={true}` will only show the animation after a default delay of `3000ms`. By its name, a consumer might confuse it to show the pulsating animation immediately and report this as a bug.
```

Comment:
What's the reasoning behind `3000ms` delay? As a user, I'd think the website is slow too if any animation triggers after a delay. Is there a problem with starting the animation immediately?

Reply:
The pulsating animation starts after 3s. This is to indicate that some progress is happening. 3s is a long duration for any process to happen on our products. For most cases, this will not show up but for slow process that happen longer than 3s, we need to give some feedback to indicate progress